### DISCUSSION TITLE
admin/diagnostics Page not found!

### DISCUSSION THREAD

**Discussion by Hardexit** [dani-garcia/vaultwarden#2832] [Q&A] [answered]
### Subject of the issue
I switched vaultwarden from docker to docker-compose and now I get the message Page not found on the admin/diagnostics page!
All other pages in the admin area work.

### Deployment environment

* vaultwarden version: 1.26.0

* Install method: docker-compose

* Clients used:  web vault

* Reverse proxy and version:  nginx 1.18.0

* MySQL/MariaDB or PostgreSQL version: 

* Other relevant details:

### Steps to reproduce

call admin page
log in
select diagnostics

### Troubleshooting data

![image](https://user-images.githubusercontent.com/112691363/196155989-6ba05008-f71b-40db-9ce2-2a13d7ed54ff.png)

**Comment by BlackDex** [comment, marked as answer]
It works fine for me on two different environments.
Be sure to first login via `/admin` using the token, else you will get a 404 page.
After you are logged in you can visit the diagnostics page.

Also, if you browser clears cookies or sessions or has some other extensions which block or clear headers those could cause some issues also.

**Reply by Hardexit** [reply]
This gave me the idea to restart the web server once, since then it works.

Yes I don't use caddy because I haven't figured out yet how to include the existing certificate there.
